tools/autograd/gen_autograd.py

```diff
@@ -23,7 +23,15 @@
 
 import argparse
 import os
+# Favoring from...to instead of import...as
```

Comment:
Why this comment here?
I understand the change in the other files, but I'm not sure having just one comment here is useful?

Reply:
Yea I was just thinking adding a note somewhere why we changed it but couldn't find a good place. Actually I will just remove this. Few people will wonder about from...to I think :D